src/components/InteractiveToastView.tsx

```diff
@@ -1,18 +1,23 @@
 import React, { useMemo } from 'react';
 import { PropsWithChildren } from 'react';
+import { screenWidth } from '../helpers/toast-helpers';
+import { CommonToastProps, ToastOptions } from '../Toast';
 import {
   GestureResponderEvent,
   PanResponderGestureState,
   ViewStyle,
+  Animated,
+  PanResponder,
+  I18nManager,
 } from 'react-native';
-import { Animated } from 'react-native';
-import { screenWidth } from '../helpers/toast-helpers';
-import { CommonToastProps, ToastOptions } from '../Toast';
-import { PanResponder } from 'react-native';
+
+const { isRTL } = I18nManager;
```

Comment:
Will is receive updates from I18nManager in case of changing from RTL to non-RTL language?

Reply:
Sure